Islandora/islandora_solution_pack_compound
# 7.x Documentation update preparing for 1.12 release

**JIRA Ticket**: None

# What does this Pull Request do?

Updates this module's documentation and moves attached media to its own relative folder.
Note: will create release version once this one is approved/corrected. (after lunch!)

# What's new?

- Updated the general setting screenshot 
- Mention some new options
- Mention that Solr is a soft dependency and explain why?
- Removed the red box around the JAIL option in the general settings screenshot. Never understood why we should mark that one only since we have at least 8 more to explain. Better to general in that case
- Changed some wording

# How should this be tested?

Proofread, check Screenshot, see if my words make sense, correct me if i'm wrong

# Interested parties
@manez @bryjbrown @bondjimbond
